### PR TITLE
Omni rules to ocs

### PR DESCRIPTION
**Pull request by sourabh-nanoti**
## Summary

1. The changes now include the rating groups configured for omnipresent rules  

## Test Plan

Integ_test for testing the change is added in the changes. 

## Additional Information

- [ ] This change is backwards-breaking -- NO

### CODE REVIEW

File: cwf/gateway/integ_tests/Makefile
Comment (themarwhal) [inline cwf/gateway/integ_tests/Makefile:15]: Don't think we need this because we only have a couple of tests for omni

File: cwf/gateway/integ_tests/Makefile
Comment (themarwhal) [inline cwf/gateway/integ_tests/Makefile:47]: Same here, don't think we need this.

File: cwf/gateway/fabfile.py
Comment (themarwhal) [inline cwf/gateway/fabfile.py:41]: Let's get rid of this too. We have multiple ways of running specific tests. One is the `tests_to_run` tag and the other is `test_re`. The `test_re` tag uses regex to match tests names. So in this case, `fab integ_test:test_re=TestOmni`, should do the trick.

**Review by themarwhal — CHANGES_REQUESTED** [review]
Can we add a unit test in FeG for this feature? And I don't think we need an entire integration test for this feature. We should modify the existing `TestOmnipresentRules` test so that it covers the OCS case as well. The integration tests take long so, it is expensive.

**Comment by uri200** [comment]
Please, dont land this, I am working on this feature

**Comment by uri200** [comment]
Sorry @sourabh-nanoti , but I was busy with some other stuff.

I will start working on this feature next week. Your change looks good but I need to add some more stuff. So lets merge this after I have my changes done

What it would be great is if you could try to see if the new test can be included in the current test. Or even better, do we really need the old test anymore? does that old tests provide any value? Maybe the new test you just created included the old test. 

See that one of the problems we have with the integ test is the time it takes, so if we can merge some test that save us some time.

File: cwf/gateway/integ_tests/omni_rules_test.go
Comment (uri200) [inline cwf/gateway/integ_tests/omni_rules_test.go:130]: Can you please try to use the other tests to test this if possible?

File: cwf/gateway/integ_tests/Makefile
Comment (uri200) [reply cwf/gateway/integ_tests/Makefile:15]: lets remove this and make OMNI part of all tests

File: feg/gateway/services/session_proxy/servicers/session_controller.go
Comment (uri200) [inline feg/gateway/services/session_proxy/servicers/session_controller.go:101]: remove this comment. This doesnt apply anymore

File: feg/gateway/services/session_proxy/servicers/session_controller.go
Comment (uri200) [inline feg/gateway/services/session_proxy/servicers/session_controller.go:106]: instead of moving this line, move line 102-104 to line 99 (right after `gx.ParseRuleInstallAVPs`). The reason is this way we will have all the rules collection logic at the same block and all the key, event and timers logic on a different block

**Comment by uri200** [comment]
As the PR rules mention, please add [feg] at the beginning of the title of the PR so we can filter it out in the futre

**Comment by uri200** [comment]
Closing this since this was covered with `injectOmnipresentRules` function on feg

**Pull request closed without merging**
